package.json

```diff
@@ -119,6 +119,12 @@
           ],
           "default": null,
           "scope": "resource"
+        },
+        "vitest.showFailMessages": {
+          "description": "Get instant feedback when using Watch Mode. Pop-ups an error when a test fails.",
+          "type": "boolean",
+          "scope": "window",
```

Comment:
note: changing this to window to prevent users from incorrectly trying to set this value at the workspace folder level